debian/rules

```diff
@@ -8,10 +8,10 @@ endif
 # Use the dpkg version comparison algorithm (which is easily in reach) to compare the build
 # system version to the version from which python3-distutils is required (18.04 and higher),
 # and conditionally populate a substitution variable which is referenced in the control file.
-# On bionic, where Ubuntu's python3.7 is used, this is a virtual package for python3-distutils
-# (in bionic-updates). On jammy, this is a real package providing distutils for python3.7 from deadsnakes.
+# On bionic, where Ubuntu's python3.8 is used, this is a virtual package for python3-distutils
+# (in bionic-updates). On jammy, this is a real package providing distutils for python3.8 from deadsnakes.
 ifeq ($(shell (. /etc/os-release && dpkg --compare-versions $$VERSION_ID "ge" "18.04" && echo yes || echo no)),yes)
-    extra_substvars = -VbionicOrLater:Depends="python3.7-distutils"
+    extra_substvars = -VbionicOrLater:Depends="python3.8-distutils"
```

Comment:
i think we might be able to drop these shenanigans now and unconditionally include distutils in the control file, no?

Reply:
yes, you're right, thanks for the catch!